.github/workflows/ci-pr-prerelease.yml

```diff
@@ -50,7 +50,7 @@ jobs:
 
             # Upload code coverage data
             - name: 'Upload code coverage for LWC to Codecov.io'
-              uses: codecov/codecov-action@v1
+              uses: codecov/codecov/codecov-action@v2.0.2
```

Comment:
The action name is incorrect in all files (you have an extra `codecov` in the path): it should be `codecov/codecov-action@v2.0.2`

Reply:
That is odd - sorry about that. I can update and resubmit.

Reply:
Regarding the test file, I only have what they told me in the case that I opened with them:

> Hi David,
> Can you please delete all `'test-result-<ID NUMBER>.json'` files prior to upload? This should get you working for now while we work to add support for the new format.
> Ticket: https://codecov.freshdesk.com/helpdesk/tickets/5620
> Best,
> Joe Becher (she/her)
> Lead Support Engineer | Codecov

Reply:
Got it, thanks.